lib/perl/Genome/WorkflowBuilder/Command.pm

```diff
@@ -90,22 +91,42 @@ sub _get_ptero_execute_method {
 
     my $self = shift;
     my $log_dir = shift;
+    my $ptero_lsf_parameters = $self->_get_ptero_lsf_parameters();
+    $ptero_lsf_parameters->{command} = sprintf(
+        'genome ptero wrapper --command-class %s '
+        .'--method execute --log-directory %s',
+        $self->command, $log_dir);
```

Comment:
I initially shared @tmooney's concern, but I eventually decided that I'm ok with these api's being different.

The api's of the job services are different because the underlying api's which the different job services expose as web services are different.  Shell-command exposes execvp(2) via Pythons subprocess.Popen().  The lsf service exposes lsb_submit(3).  The workflow service makes few demands as possible on the api's that the job services expose.  This allows us to keep the definition of what a job service can do as a broad as possible.

If we have stricter requirements on the api which a job service must expose, the we limit the kinds of job services which we could eventually offer.